testutils/clustermanager/gke_test.go

```diff
@@ -29,43 +33,104 @@ var (
 	fakeCluster = "d"
 )
 
```

Comment:
I could swear that I saw `TestSetup()` somewhere...

Reply:
I probably should rename that method to `setupForTest`, that's for setting up each test